modules/appeals_api/config/schemas/v2/200995.json

```diff
@@ -86,45 +87,41 @@
                     "retrieveFrom": {
                       "type": "array",
                       "items": {
-                        "type": "array",
-                        "items": {
-                          "type": "object",
-                          "properties": {
-                            "type": { "type": "string", "enum": ["retrievedEvidence"] },
-                            "attributes": {
-                              "type": "object",
-                              "properties": {
-                                "locationAndName": { "$ref": "#/definitions/nonBlankString" },
-                                "evidenceDates": {
-                                  "type": "array",
-                                  "items": { "$ref": "#/definitions/date" },
-                                  "minItems": 1
-                                }
-                              },
-                              "additionalProperties": false,
-                              "required": [ "locationAndName", "evidenceDates" ]
-                            }
-                          },
-                          "additionalProperties": false,
-                          "required": [ "type", "attributes" ]
+                        "type": "object",
+                        "properties": {
+                          "type": { "type": "string", "enum": ["retrievedEvidence"] },
+                          "attributes": {
+                            "type": "object",
+                            "properties": {
+                              "locationAndName": { "$ref": "#/definitions/nonBlankString" },
+                              "evidenceDates": {
+                                "type": "array",
+                                "items": { "$ref": "#/definitions/date" },
+                                "minItems": 1
+                              }
+                            },
+                            "additionalProperties": false,
+                            "required": [ "locationAndName", "evidenceDates" ]
```

Comment:
Are the `evidenceDates` here supposed to align with the form? On the form it's called `date(s) of record`. Just want to be sure we're sure, and that I'm actually understanding this!

Reply:
Hmm, that is a good question. This is just based on the initial draft schema that Jack created... I do like the idea of keeping the language as close to what is on the form as possible.

Reply:
It's equivalent to the dates needed when requesting evidence. I suppose we could rename it to `dateOfRecord`.

Reply:
I'd say `dateOfRecord` or `evidenceDate` <- singular since each piece of evidence has one date. Thoughts?

Reply:
Unfortunately, that's not how this works. The form has a list of `locationAndName`s, and then a list of evidence date(s) to be retrieved. We're not able to require singular, given the way the form is structured.

Reply:
ok, so then would it be appropriate to say `locationsAndNames` and `datesOfRecord` ?

Reply:
I don't think `locationAndNames` needs to be plural - but yes to `datesOfRecord`